src/image-tags.js

```diff
@@ -54,6 +54,34 @@ function pickImage(pageType, data, url) {
   }
 }
 
+function pickAttributionFromCard (story, cardId) {
+  const { metadata = {} } = story.cards.find(card => card.id === cardId) || {};
+  if(metadata && !isEmpty(metadata) && get(metadata, ['social-share', 'image', 'attribution'], false)) {
```

Comment:
while card level share, taking `attribution` from each card, if it's not present, then it will fallback to story attribution logic.